my_list.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(arr)
+    @list = arr
+  end
```

Comment:
- [ ] When I run the project with the sample test command, it returns an error indicating that, expected 1 argument and passed 3. In your initialize method, it should take a list of arguments and not only 1.

- You could have done something in this way:
```ruby
    def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```

- [ ] Kindly work on that 👍 

**See the error screenshot below**
![image](https://user-images.githubusercontent.com/50941074/199969734-4f0c3500-9c1a-46f2-a8c7-82451842f83b.png)

If you run this test command below:
`ruby -r "./my_list.rb" -e "p MyList.new(1,2,4)"`

You should get this result
`<MyList: @list=[1, 2, 4]>`

Reply:
@AmaduKamara Thanks for pointing that out we were looking for that solution only which accepts n args instead of 1